torch/_inductor/codegen/cpp_micro_gemm.py

```diff
@@ -514,11 +514,9 @@ class CppMicroGemmAMX(CppMicroGemm):
 {%- if use_cached_dequantized_B %}
     // Create a stack-allocated buffer for tiles of B.
     // Except maybe for the tail-case, an AMX tile of B has 16x32 BF16 elements.
-    const auto num_elements_per_b_tile = 512;
-    const auto last_k_offset = K / {{block_k}} * {{block_k}};
-    const auto tail_k_size = K - last_k_offset;
     // we cache K * {{block_n}} elements of dequantized B
-    const auto buf_size = K * {{block_n}} * sizeof({{input_t}});
+    const auto block_n = {{block_n}};
```

Comment:
nit: why not use `{{block_n}}` directly?

Reply:
I didn't do that because of style concerns - then only one argument to the API call of the inner-kernel would have not been a variable.

As Jiong mentioned, I'll create another variable & replace argument name `ldb` with it.